_seattle/API Chaya.md

```diff
@@ -1,18 +1,18 @@
 ---
 name: API Chaya
-address: API Chaya, P.O. Box 14047, Seattle, WA 98114
+address: 
 lat: 
```

Comment:
It looks like not all of the resources have lat/long values in their yaml files? If not, we should probably manually fill them in.